### PR TITLE
Thematic collections facet

### PR DESCRIPTION
**Pull request by mirjamverloop**
- The facet component can handle radio buttons as well.
- The actual themes are hard-coded for now. 
- The theme facet is the first one in the column.

### CODE REVIEW

File: pages/search/index.vue [hunk 1]
```diff
@@ -141,6 +142,21 @@
     };
   }
 
+  const thematicCollections = [ { label: 'All items', value: '' },
```
Comment (rwd) [inline pages/search/index.vue]: The labels should go into lang/en.js

File: components/search/SearchSelectedFacets.vue [hunk 2]
```diff
@@ -24,8 +24,16 @@
       facetList: function() {
         let listOfFacets = [];
         for (let facetName in this.facets) {
+
```
Comment (rwd) [inline components/search/SearchSelectedFacets.vue]: In this component, the filter badge text should use the _label_ of the theme, not its query value, e.g. "1914-1918" not "ww1"

File: components/search/SearchSelectedFacets.vue [hunk 3]
```diff
@@ -7,7 +7,12 @@
       class="mr-2"
       data-qa="filter badge"
     >
-      {{ $t('formatting.labelledValue', { label: $t(`facets.${selectedFacet.facetName}`), value: selectedFacet.fieldValue}) }}
+      <template v-if="selectedFacet.facetName === 'THEME'">
```
Comment (rwd) [inline components/search/SearchSelectedFacets.vue]: Let's make this more reusable by switching to:
```js
<template v-if="$te(`facets.${selectedFacet.facetName}.options`)">
```

Comment (mirjamverloop) [reply components/search/SearchSelectedFacets.vue]: Good suggestion, thanks. I have updated the code. 

I might add a test with options as well, and update the mocked $te.

**Pull request merged**